crates/router/src/routes/app.rs

```diff
@@ -561,6 +561,9 @@ impl Payments {
                     .route(web::post().to(payments::payments_create_intent)),
             );
 
+        route = route.service(
+            web::resource("/aggregate").route(web::get().to(payments::get_payments_aggregates)),
+        );
```

Comment:
Nit: You can combine with the previous routes being registered.

```suggestion
            )
            .service(
                web::resource("/aggregate").route(web::get().to(payments::get_payments_aggregates)),
            );
```